src/HDInsightOnAks/HdInsightOnAks.Autorest/examples/New-AzHdInsightOnAksCluster.md

```diff
@@ -0,0 +1,358 @@
+
+$clusterType="Trino"
+# Get available cluster version based the command Get-AzHdInsightOnAksAvailableClusterVersion
+$clusterVersion= (Get-AzHdInsightOnAksAvailableClusterVersion -Location $location | where-object {$_.ClusterType -eq $clusterType})[0]
+
+# user msi related parameters
+$msiResourceId="/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/{your resource group name}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{your msi}";
+$msiClientId="00000000-0000-0000-0000-000000000000";
+$msiObjectId="00000000-0000-0000-0000-000000000000";
+
+# cluster authorization information
+$userId="00000000-0000-0000-0000-000000000000";
+
+# create node profile
+$vmSize="Standard_E8ads_v5";
+$workerCount=5;
+
+$nodeProfile = New-AzHdInsightOnAksNodeProfileObject -Type Worker -Count $workerCount -VMSize $vmSize
+
+$keyVaultResourceId="/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/{your resource group name}/providers/Microsoft.KeyVault/vaults/{your vault name}";
+$secretName="{your secret name}"
+$referenceName="{your secret reference name}";
+
+$secretReference=New-AzHdInsightOnAksSecretReferenceObject -SecretName $secretName -ReferenceName $referenceName
+
+#hive catalog configuration
+
+$catalogName="{your catalog name}"
+$metastoreDbConnectionURL="jdbc:sqlserver://{your sql server url};database={your db name};encrypt=true;trustServerCertificate=true;loginTimeout=30;";
+$metastoreDbUserName="{your db user name}";
+$metastoreDbPasswordSecret=$secretName;
+$metastoreWarehouseDir="abfs://{your container name}@{your adls gen2 endpoint}/{your path}";
+
+$trinoHiveCatalogOption=New-AzHdInsightOnAksTrinoHiveCatalogObject -CatalogName $catalogName -MetastoreDbConnectionUrl $metastoreDbConnectionURL -MetastoreDbConnectionUserName $metastoreDbUserName -MetastoreDbConnectionPasswordSecret $metastore{your secret name}Secret -MetastoreWarehouseDir $metastoreWarehouseDir
+
+$clusterName="{your cluster name}";
+
+
+New-AzHdInsightOnAksCluster -Name $clusterName `
+                            -PoolName $clusterPoolName `
+                            -ResourceGroupName $resourceGroupName `
+                            -Location $location `
+                            -ClusterType $clusterType `
+                            -ClusterVersion $clusterVersion.ClusterVersionValue `
+                            -OssVersion $clusterVersion.OssVersion `
+                            -AssignedIdentityResourceId $msiResourceId `
+                            -AssignedIdentityClientId $msiClientId `
+                            -AssignedIdentityObjectId $msiObjectId `
+                            -ComputeProfileNode $nodeProfile `
+                            -AuthorizationUserId $userId `
+                            -KeyVaultResourceId $keyVaultResourceId `
+                            -SecretReference $secretReference `
+                            -TrinoHiveCatalog $trinoHiveCatalogOption
+```
+
+```output
+{{ Add output here }}
```

Comment:
Is it expected to no output here? if yes, please remove output block. Notice that please check other blocks as well.

Reply:
Fixed.